core/src/main/java/org/infinispan/conflict/impl/EntryMergePolicyFactory.java

```diff
@@ -0,0 +1,22 @@
+package org.infinispan.conflict.impl;
+
+import org.infinispan.conflict.EntryMergePolicy;
+import org.infinispan.factories.AbstractNamedCacheComponentFactory;
+import org.infinispan.factories.AutoInstantiableFactory;
+import org.infinispan.factories.annotations.DefaultFactoryFor;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.1
+ */
+@DefaultFactoryFor(classes = EntryMergePolicy.class)
+public class EntryMergePolicyFactory extends AbstractNamedCacheComponentFactory implements AutoInstantiableFactory {
```

Comment:
I created this for convenience as much as anything, I'm happy to remove it and just retrieve from the configuration